src/qt/treectrl.cpp

```diff
@@ -8,11 +8,438 @@
 // For compilers that support precompilation, includes "wx.h".
 #include "wx/wxprec.h"
 
+#ifdef __BORLANDC__
+#pragma hdrstop
+#endif
+
 #include "wx/treectrl.h"
+#include "wx/imaglist.h"
+
+#include "wx/qt/private/winevent.h"
+#include "wx/qt/private/treeitemfactory.h"
 
 #include <QtWidgets/QTreeWidget>
+#include <QtWidgets/QHeaderView>
+#include <QtGui/QPainter>
+
+namespace
+{
+struct TreeItemDataQt
+{
+    TreeItemDataQt() : data(NULL)
+    {
+    }
+
+    TreeItemDataQt(wxTreeItemData* data) : data(data)
+    {
+        if ( !registered )
+        {
+            qRegisterMetaTypeStreamOperators<TreeItemDataQt>("TreeItemDataQt");
+            registered = true;
+        }
+    }
+
+    wxTreeItemData *data;
+
+    private:
+    static bool registered;
```

Comment:
Also, in wx conventions static should be easily greppable for by their special prefix:
```suggestion
    static bool s_registered;
```